ReadMe_PMT.md

```diff
@@ -64,5 +64,44 @@
 
     - `'board_gem_channels'	: [5,6,7]`
 
+#### Note: Both axis of these waveforms are shown in the raw form, meaning _x_ is _samples_, and _y_ is _ADC counts_. The transformation to nanoseconds and millivolts is not automatically made.
 
+## Digitizers relevant information:
 
+The CYGNO DAQ system uses two digitizers, typically designated as *fast* and *slow* board/digitizer.
+
+The intent is to be able to save signals with both small and large time extensions - fast and slow, respectively.
+
+- Examples of *short/fast* signals are low energy (<20 keV) electron recoils (e.g. the 55Fe X-rays at 6 keV). High energy and high energy deposition particle (such as alpha particles), can also be contained in time most of the times.
+
+- Examples of *long/slow* signals would be higher energy cosmics or electron recoils (>100 keV and/or MIPs).
+
+### Models and settings currently used:
+
+- **Fast digitizer:** CAEN V1742 
+    - https://www.caen.it/products/v1742/
+    - 12 bits
+        - ADC counts are within [0,4096]
+    - Dynamic range: 1Vpp
+    - Sampling: 750Mhz
```

Comment:
If you want to add relevant details you could add that the lowest voltage reachable by the  digitizer can vary (in LIME waveforms can go from -0.8 to 0.2 V I think, but needs to be checked)

Also the sampling can be modified (in MANGO I think we always use 1 GS/s). So I would write it and say that for LIME it is 750 MS/s

Reply:
I confirmed the DC_offset for the used digitizers is [-1,+1] Volts, and that in all detectors we use -0.3. Higher values increase the "incorrections" in the waveforms, making them harder to correct a posteriori. Comments added.

I confirmed all detectors use 750Mhz. Added this info together with variable sampling information.